src/components/LatestRatings.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { gql } from 'apollo-boost';
+import { Query } from 'react-apollo';
+
+const Hero = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  min-height: calc(100vh - 7.375rem);
+  text-align: center;
+`;
+
+const GET_LATEST_RATINGS_QUERY = gql`
+  query GET_LATEST_RATINGS_QUERY($max: Int) {
+    getLatestRatings(max: $max) {
+      _id
+      author
+      caller
+      callType
+      message
+      score
+      createdAt
+      number {
+        number
+      }
+    }
+  }
+`;
+
+function Comment({ rating }) {
+  return (
+    <li
+      style={{
+        backgroundColor: '#e8f5e9',
+        margin: '1rem',
+        padding: '1rem',
+      }}
+      key={rating._id}
+    >
+      <h3
+        style={{
+          borderBottom: '1px solid rgba(0, 0, 0, 0.1)',
+          fontWeight: 300,
+          margin: 0,
+        }}
+      >
+        {rating.author} reported this number as "{rating.callType}"
+      </h3>
+      <p>{rating.message}</p>
+      <time style={{ fontSize: '0.75rem' }}>
```

Comment:
You should add `datetime` for the `<time>` tag. `<time datetime={....date... toISOString()}`> would be fine.

Reply:
i added but it throw an error 
my code
` <time dateTime={rating.createdAt.toISOString()} style={{ fontSize: '0.75rem' }}>`
error 
`TypeError: rating.createdAt.toISOString is not a function`

Reply:
and can you please tell me why it is needed? because date already being shown.

`      <time style={{ fontSize: '0.75rem' }}>
            {new Date(parseInt(rating.createdAt)).toLocaleString('en-GB')}
      </time>
`

Reply:
We need to provide a machine readable format so we can change the displayed time however we want. If we display it as `2h ago` (or in ay other language) and provide `datetime` in the `time` tag, machines will still understand what it means.

Anyway, you need to create a new `Date` object from `rating.createdAt` so you can convert it to an ISO string.

Reply:
solved in latest commit